vortexasdk/utils.py

```diff
@@ -20,3 +23,29 @@ def filter_empty_values(data: Dict) -> Dict:
     return {
         k: v for k, v in data.items() if not (v is None or v == [] or v == {})
     }
+
+
+def get_latest_package_version(package_name: str):
+    """
+    Retrieves the latest SDK version from PyPI and compares with the currently installed SDK version.
+    """
```

Comment:
We currently only ever call this method with the argument `(from vortexasdk import __name__ as pkg_name)`, so we could do away with the argument, and call it `get_latest_sdk_version`. This makes the code simpler.